custom_commands/suppressedNotificationsTrueCheck.js

```diff
@@ -0,0 +1,70 @@
+const axios = require('axios');
+
+module.exports = {
+    command: async function () {
+
+        const getToken = async () => {
+            const response = await axios.post(
+                'https://qa2.development.tidepool.org/auth/login',
+                '',
+                {
+                    headers: {
+                        'Authorization': 'Basic YnJpYW4rdXBsb2FkNzg2d2ViMTMyN2NsaW5pYzM1YUB0aWRlcG9vbC5vcmc6dGlkZXBvb2w=',
```

Comment:
Let's not include credentials in PRs that are public to the world. This should come from injected environment variable at run-time (i.e. in the CI).

Reply:
changed the password for this account for now. will look into injecting credentials into the environment variable.